cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -34,9 +36,21 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
       p("image::grant-privileges-graph.png[title=\"GRANT and DENY Syntax\"]")
     }
     section("Listing privileges", "administration-security-subgraph-show") {
-      p("Available privileges for a particular role can be seen using `SHOW  PRIVILEGES`.")
+      p("Available privileges for all roles can be seen using `SHOW  PRIVILEGES`.")
+      query("SHOW PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
+        p("Lists all privileges for all roles")
+        resultTable()
+      }
+
+      p("Available privileges for a particular role can be seen using `SHOW ROLE $role PRIVILEGES`.")
       query("SHOW ROLE regularUser PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
-        p("Lists all privileges")
+        p("Lists all privileges for role 'regularUser'")
+        resultTable()
+      }
+
+      p("Available privileges for a particular user can be seen using `SHOW USER $role PRIVILEGES`.")
```

Comment:
`SHOW USER **$role** PRIVILEGES` Should probably be user

Reply:
shoudl there even be a $ here (and for roles)? It is not a parameter that should be supplied...

Reply:
I know that the example queries for this in the role and user management sections use `SHOW USER/ROLE name PRIVILEGES`